### PR TITLE
Initial release

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2022 Toitware ApS.
```
Comment (kasperl) [inline CMakeLists.txt]: 2023?

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Publish package
```
Comment (kasperl) [inline .github/workflows/publish.yml:16]: Copyright?

File: tests/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2022 Toitware ApS. All rights reserved.
```
Comment (kasperl) [inline tests/CMakeLists.txt]: 2023?

File: tools/toit.cmake [hunk 4]
```diff
@@ -0,0 +1,118 @@
+# Copyright (C) 2019 Toitware ApS.
```
Comment (kasperl) [inline tools/toit.cmake]: 2023?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: .github/workflows/publish.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/publish.yml:16]: done.

File: CMakeLists.txt (hunk 1 above)
Comment (floitsch) [reply CMakeLists.txt]: done.

File: tests/CMakeLists.txt (hunk 3 above)
Comment (floitsch) [reply tests/CMakeLists.txt]: done.

File: tools/toit.cmake (hunk 4 above)
Comment (floitsch) [reply tools/toit.cmake]: done.

**Pull request merged**